### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by tomassrnka**
Update Slack link for Docker Community Slack

**- What I did**
- Found broken link

**- How I did it**
- Read CONTRIBUTING.md

**- How to verify it**
- Click on the link :) 

**- Description for the changelog**
- Updated the link to the correct one

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -101,7 +101,7 @@ the contributors guide.
     <td>
       <p>
         Register for the Docker Community Slack at
-	<a href="https://community.docker.com/registrations/groups/4316" target="_blank">https://community.docker.com/registrations/groups/4316</a>.
+	<a href="http://dockr.ly/slack" target="_blank">http://dockr.ly/slack</a>.
```
Comment (tianon) [inline CONTRIBUTING.md:104]: I think this should probably still be `https`, right?

```suggestion
	<a href="https://dockr.ly/slack" target="_blank">https://dockr.ly/slack</a>.
```

**Review by tianon — APPROVED** [review]
LGTM :+1:

**Review by samuelkarp — APPROVED** [review]
LGTM

**Review by cpuguy83 — DISMISSED** [review]
LGTM

**Comment by cpuguy83** [comment]
Looks like commit is missing DCO
Can you also squash the commits?

**Comment by neersighted** [comment]
Change is already in master

**Pull request closed without merging**